packages/browser-wallet/src/popup/shared/TransactionReceipt/displayPayload/DisplayUpdateContract.tsx

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { UpdateContractPayload } from '@concordium/web-sdk';
 import { displayAsCcd } from 'wallet-common-helpers/lib/utils/ccd';
+import { SmartContractParameters } from '@shared/utils/types';
 
 interface Props {
     payload: UpdateContractPayload;
-    parameters?: Record<string, unknown>;
+    parameters?: SmartContractParameters;
```

Comment:
This component (not this line, but line 30) gives an error trying to read `toString` of undefined, when I tried to transfer wCCD tokens

Reply:
It should be fixed now.
FYI: 865 does not work, you should add wccd using 866